### PR TITLE
topology/cmake: skip all topologies when alsatplg < 1.2.5

### PR DESCRIPTION
**Pull request by marc-hb**
Stop producing corrupted .tplg files when using `codec_consumer` (e.g.:
sof-imx8mp-btsco-dual-8ch.tplg, sof-imx8ulp-9x9-btsco-16k.tplg, ...)

Then we'll be able to finally search/replace "codec_master", see revert

### CODE REVIEW

File: tools/topology/CMakeLists.txt [hunk 1]
```diff
@@ -25,6 +25,25 @@ ${stdout}")
 endfunction()
 
 
+# Being written in C, `alsatplg` silently ignores some invalid inputs
+# and produces an corrupt .tplg file instead of returning an error code
+# that fails the build.  For instance, alsatplg versions < 1.2.5 silently
+# corrupt `codec_consumer` and turn it into `codec_master` instead.
```
Comment (cujomalainey) [inline tools/topology/CMakeLists.txt:31]: nit: some grammer tense issues

**Review by cujomalainey — APPROVED** [review]

File: tools/topology/CMakeLists.txt (hunk 1 above)
Comment (marc-hb) [reply tools/topology/CMakeLists.txt:31]: Thanks for the prompt review!

Could you be more precise about the grammar issues?

I found "produces A corrupt..." and nothing else (and that's not a tense). Google docs and MS Word don't find anything either.

**Comment by marc-hb** [comment]
https://sof-ci.01.org/sofpr/PR8377/build14077/devicetest and https://sof-ci.01.org/sofpr/PR8377/build14078/devicetest are 100% green, even suspend/resume. Suspicious!

**Review by lgirdwood — APPROVED** [review]
@dbaluta fyi, this should help with some imx topologies.

**Review by kv2019i — APPROVED** [review]

**Pull request merged**